admin-core/src/main/resources/application.conf

```diff
@@ -75,18 +75,20 @@ drill {
         lifetime = ${?DRILL_JWT_LIFETIME}
     }
     password {
-        minLength = ${?DRILL_PASSWORD_MIN_LENGTH}
         minLength = 6
-        mustHaveUppercase = ${?DRILL_PASSWORD_HAVE_UPPERCASE}
+        minLength = ${?DRILL_PASSWORD_MIN_LENGTH}
         mustHaveUppercase = false
-        mustHaveLowercase = ${?DRILL_PASSWORD_HAVE_LOWERCASE}
+        mustHaveUppercase = ${?DRILL_PASSWORD_HAVE_UPPERCASE}
         mustHaveLowercase = false
-        mustHaveDigit = ${?DRILL_PASSWORD_HAVE_DIGIT}
+        mustHaveLowercase = ${?DRILL_PASSWORD_HAVE_LOWERCASE}
         mustHaveDigit = false
+        mustHaveDigit = ${?DRILL_PASSWORD_HAVE_DIGIT}
```

Comment:
were these properties just shuffled around?

Reply:
env should come after default otherwise it'll be overwritten with default anyway